### DISCUSSION TITLE
kowl permission for kafka3.x craft w authorization

### DISCUSSION THREAD

**Discussion by zzaman016** [community/community#126367] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone! please tell me what kowl permission are required to work with kafka 3.x craft and authorization?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @zzaman016 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
